### DISCUSSION TITLE
Two factor

### DISCUSSION THREAD

**Discussion by 9228262828** [community/community#141452] [Other Feature Feedback, Questions, & Ideas] [answered]
I had lost my two factor account 
And now i can't sign in to my account in GitHub what should i do 
I need to make support disable this method

**Comment by queenofcorgis** [comment, marked as answer]
Hi @9228262828, 

Thank you for your patience. The community is unable to assist with this situation, please refer to our [Recovering your account documentation ](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) and virtual assistant within that document to aid you in this situation. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
